acquia_lift.module

```diff
@@ -2853,35 +2778,16 @@ function acquia_lift_command_messagebox($message, $seconds = 0) {
 }
 
 /**
- * Returns an AJAX command to toggle the page variation element selector.
+ * Returns an AJAX command to toggle the element variation element selector.
  *
  * @param bool $start
- *   True to start the page variation mode, FALSE to stop it.
+ *   True to start the element variation mode, FALSE to stop it.
  */
-function acquia_lift_command_page_variation_toggle($start = TRUE) {
+function acquia_lift_command_element_variation_toggle($start = TRUE) {
   return array(
     'command' => 'acquia_lift_variation_toggle',
     'data' => array(
       'start' => $start,
-      'type' => 'page',
-    ),
-  );
-}
-
-/**
- * Returns an AJAX command to preview a specific page variation.
- *
- * @param string $agent_name
- *   The name of the campaign/agent to which the variation belongs.
- * @param int $variation_index
- *   The index of the variation to preview.
- */
-function acquia_lift_command_page_variation_preview($agent_name, $variation_index) {
```

Comment:
Ya it's always the cache.

On the other hand, I read the code and think the function acquia_lift_command_element_variation_preview (originally acquia_lift_command_page_variation_preview) and its if switch was AB campaign only, so I have [removed it](https://github.com/acquia/acquia_lift/commit/a5cb54eb0c75ba62cbb2f67ab684c7dc32ee6ffb).